kolibri_dynamic_collections_plugin/assets/src/modules/collectionBase/handlers.js

```diff
@@ -1,10 +1,7 @@
 export function showCollection(store) {
-  store.commit('CORE_SET_PAGE_LOADING', true);
-
   return Promise.all([store.dispatch('setAllChannelInfo')])
     .then(() => {
-      store.commit('CORE_SET_PAGE_LOADING', false);
-      store.commit('CORE_SET_ERROR', null);
+      // nothing to do...
```

Comment:
I think just remove the `then`, right? The promise still needs to be resolved in the caller.